### ISSUE TITLE
IPV6 mac resolution not working 

### ISSUE THREAD

**Issue by cj-andras-tevesz** [GhostofGoes/getmac#84]
**Describe the bug**
IPV6 mac resolution is not working for link-local addresses.

**To Reproduce**

test@ubuntu:$ getmac --version
getmac 0.9.2
test@ubuntu:$ **getmac** -6 "fd89:6ddf:345a:0:1c32:f2a:c402:3a79"
test@ubuntu:$ **ndisc6** "fd89:6ddf:345a:0:1c32:f2a:c402:3a79"  wlan0
Soliciting fd89:6ddf:345a:0:1c32:f2a:c402:3a79 (fd89:6ddf:345a:0:1c32:f2a:c402:3a79) on wlan0...
Target link-layer address: A8:51:AB:0F:36:14
 from fe80::465:dbcf:6472:189c

**Expected behavior**
IPV6 mac resolution for link-local addresses.
Call ndisc6 https://www.remlab.net/ndisc6/ to do the discovery. 

**System info**
(please complete the following information):
 -  Ubuntu 22.04.2 LTS
 - Python 3.10.6
 - getmac 0.9.2

**Comment by GhostofGoes** [comment]
Yup, this is definitely a bug. ARP is v4-only, not sure why I marked it as supporting both v4 and v6. Fix is inbound.

**Comment by GhostofGoes** [comment]
Give [0.9.3](https://github.com/GhostofGoes/getmac/releases/tag/0.9.3) a whirl. If it's still broken, please feel free to reopen the issue.
